README.md

```diff
@@ -145,6 +152,117 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 1. Node.js Client API Overview
+
+The first step is configuration. You can configure the Node.js client declaratively or programmatically. We will use the programmatic approach throughout this chapter. Please refer to the [Node.js Client Declarative Configuration section](#declarative-configuration) for details.
+
+```javascript
+var clientConfig = new Config.ClientConfig();
+clientConfig.groupConfig.name = 'dev';
+clientConfig.networkConfig.addresses.push('10.90.0.1', '10.90.0.2:5702');
+```
+
+The second step is to initialize the `HazelcastClient` to be connected to the cluster.
+
+```javascript
+Client.newHazelcastClient(clientConfig).then(function (client) {
+    // some operation
+});
+```
+
+**This client object is your gateway to access all Hazelcast distributed objects.**
+
+Let’s create a map and populate it with some data.
+
+```javascript
+var mapCustomers;
+Client.newHazelcastClient(clientConfig).then(function (client) {
+    mapCustomers = client.getMap('customers'); // creates the map proxy
+    return mapCustomers.put('1', new Customer('Furkan', 'Senharputlu'));
+}).then(function () {
+    return mapCustomers.put('2', new Customer("Joe", "Smith"));
+}).then(function () {
+    return mapCustomers.put('3', new Customer("Muhammet", "Ali"));
+});
+```
+
+As a final step, if you are done with your client, you can shut it down as shown below. This will release all the used resources and will close connections to the cluster.
+
+```javascript
+...
+.then(function () {
+    client.shutdown();
+});
+```
+
+## 2. Using Distributed Data Structures
+
+Most of the Distributed Data Structures are supported by the Node.js client. When you use clients in other languages, you should check for the exceptions.
+
+As a general rule, you configure these data structures on the server side and access them through a proxy on the client side.
+
+## 2.1. Using Map
```

Comment:
These are just samples but not individual subsections, right? I do not think they need `#` title and title number.

Reply:
I will add them to the `Table of Contents` at the end so users can access them easily.